pages/common/wcurl.md

```diff
@@ -0,0 +1,21 @@
+# wcurl
+
+> A simple wrapper around curl to easily download files.
+> See also: `curl`.
+> More information: <https://curl.se/wcurl/manual.html>.
+
+- Download the contents of a URL to a file indicated by the URL ("foo" in this case):
+
+`wcurl {{https://example.com/foo}}`
+
+- Download the contents of a URL to a file with a specified name:
+
+`wcurl {{[-o|-O|--output]}} {{bar}} {{https://example.com/foo}}`
```

Comment:
```suggestion
`wcurl {{[-o|--output]}} {{bar}} {{https://example.com/foo}}`
```

Reply:
Both shortforms are accepted, probably for wider compatibility with `wget`, unless we want to be less confusing to the user?